src/app/dashboard/components/exoft-achievements/exoft-achievements.component.html

```diff
@@ -1,28 +1,33 @@
 <div class="exoftachiv">
+
   <div class="tittle">
```

Comment:
<div class="exoftachiv"> and   <div class="tittle"> are not one-level tags. you can remove epty line here

Reply:
fixed